lib/cnet/protosw/cnet_protosw.c

```diff
@@ -14,12 +14,6 @@
 #include "cnet_protosw.h"
 #include "cne_log.h"        // for CNE_LOG, CNE_LOG_DEBUG, CNE_LOG_WARNING
 
-/**
```

Comment:
I think this is the file description? maybe we didn't mean to delete it?

Reply:
or maybe it is just for the array?

Reply:
Yes you are correct the comment is for the file.